src/builder/Accounts.sol

```diff
@@ -12,6 +12,7 @@ library Accounts {
         uint256 chainId;
         QuarkState[] quarkStates;
         AssetPositions[] assetPositionsList;
+        CometPosition[] cometPositions;
```

Comment:
`CometPositions` with an `s`?

Reply:
It feels slightly weird for the type itself to be plural. Like an array of addresses would be `address[]` not `addresses[]`.

But I guess the argument for `CometPositions` is that each instance of the type itself (potentially) contains multiple positions, like multiple accounts with multiple `base` balances?

Reply:
updated to `CometPositions`